sass/directives/_boxes.scss

```diff
@@ -10,3 +9,23 @@
   padding: $base-padding;
   background-color: $panel-primary;
 }
+
+// Accordion Panels
+@mixin accordion__panel {
```

Comment:
@toastercup I asked Elliott to switch this from extends to mixins. I haven't done my own comparison of the efficiency so I'm not wed to either approach, but I personally lean towards mixins becuase:
* The CSS is easier to read since the mixin output is part of the ruleset where it's been included and it isn't printed elsewhere in the CSS
* Mixins should be more efficient and more performant combined with compression (but I don't know if we're compressing, maybe we need a story)

There's others making these same arguments and more: 
http://csswizardry.com/2016/02/mixins-better-for-performance/
https://tech.bellycard.com/blog/sass-mixins-vs-extends-the-data/

I still prefer extends for things like buttons where we have a base button style and we just change the color or size, but not for common patterns that are used on unrelated components.